modules/ROOT/pages/kubernetes/operations/backup-restore.adoc

```diff
@@ -113,8 +143,9 @@ backup:
   databaseAdminServiceName:  "standalone-admin" #This is the Neo4j Admin Service name.
   database: "neo4j,system"
   cloudProvider: "gcp"
-  secretName: "gcpcreds"
-  secretKeyName: "credentials"
+  #secretName: "gcpcreds"
+  #secretKeyName: "credentials"
+  #gcpServiceAccountName: "gcp-sa"
```

Comment:
remove gcpServiceAccountNAme , awsServiceAccountName
and uncomment secretName and secretKeyName..same as before

Reply:
no there is no field like azureServiceAccountName

serviceAccountName is a generic field irrespective of cloud providers

the only field required for azure is azureStorageAccountName

Reply:
I understand, but the secret and a service/storage account are interchangeable. You can't use both a secret and a service/storage account, right?

Reply:
Or do you mean that in Azure, you need to configure the backup, while in gcp and aws, you don't if you have specified the service account field?

Reply:
you can use both secret and serviceAccountNAme.
You cannot use both secret and azureStorageAccountName

azureStorageAccountName is allowed only with serviceAccountName

azureStorageAccountName is the azure storage account name which contains the container where the backup files will be uploaded...this is not required for aws and gcp since in that case just bucketName is enough